packs/data/consumable-effects.db/effect-lastwall-soup-improved.json

```diff
@@ -0,0 +1,51 @@
+{
+    "_id": "5o33sch67Z8j5Vom",
+    "data": {
+        "badge": null,
+        "description": {
+            "value": "<p>Granted by @Compendium[pf2e.feats-srd.Gravelands Herbalist]{Gravelands Herbalist}</p>\n<p>Implemented effects:</p>\n<ul>\n<li>+3 item bonus to Stealth checks against undead</li>\n</ul>\n<p>Unimplemented effects:</p>\n<ul>\n<li>+3 item bonus to Stealth DCs against undead</li>\n</ul>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "hours",
+            "value": 1
+        },
+        "level": {
+            "value": 1
+        },
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "label": "Lastwall Soup (vs. undead)",
+                "predicate": {
+                    "all": [
+                        "undead"
+                    ]
+                },
+                "selector": "stealth",
+                "type": "item",
+                "value": 3
+            }
+        ],
+        "source": {
+            "value": ""
+        },
+        "start": {
+            "initiative": null,
+            "value": 0
+        },
+        "target": null,
+        "tokenIcon": {
+            "show": true
+        },
+        "traits": {
+            "custom": "",
+            "rarity": "common",
```

Comment:
```suggestion
            "rarity": "uncommon",
```

Reply:
Fixed in commit [e58fa08](https://github.com/foundryvtt/pf2e/pull/2596/commits/e58fa08ddcd4be0ff493652b4f8a9085a789fd65)